rag_env/src/rag_api.py

```
@@ -1,18 +1,12 @@
+from .paths_config import HR_POLICY_FILE_1, HR_POLICY_FILE_2, INDEX_DIR
 from fastapi import FastAPI, HTTPException, Depends
 from pydantic import BaseModel
```

Ruff (0.12.2)

`pydantic.BaseModel` imported but unused

Remove unused import: `pydantic.BaseModel`

(F401)